launch/package.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>launch</name>
+  <version>0.0.0</version>
+  <description>The ROS launch tool.</description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <license>Apache License 2.0</license>
+
```

Comment:
Not that I am aware of. It is a pure Python package and only uses Python internal modules.